collaborative/IOU/vulnerability_assessment.ipynb

```diff
@@ -0,0 +1,192 @@
+    "\n",
+    "In the cell below, we read the csv file in. We extracted three random locations from the HadISD station list as an example here -- you'll want to replace with your own locations file!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dd83d9d2-f753-4a7d-ba0b-cb30dc514b77",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Read in dummy locations from `stations_csv` file\n",
+    "from climakitae.core.paths import stations_csv_path\n",
+    "from climakitae.util.utils import read_csv_file\n",
+    "example_locs = read_csv_file(stations_csv_path, index_col=0)[['LAT_Y', 'LON_X']].rename(columns={'LAT_Y': 'lat', 'LON_X': 'lon'})"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "17a24db2-dc22-41b9-a259-682781a0b0b0",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# select a location from the location list\n",
+    "one_loc = example_locs.loc[example_locs.index == 0]\n",
+    "loc_lat = one_loc.lat.values[0]\n",
+    "loc_lon = one_loc.lon.values[0]\n",
+    "print(loc_lat, loc_lon)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "33a18588-8941-416f-b641-5dcd7d342884",
+   "metadata": {},
+   "source": [
+    "### Step 2: Retrieve metric data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ca66a74c",
+   "metadata": {},
+   "source": [
+    "Now, with the above desired points, we can pass in a series of arguments into the below `cava_data` function to retrieve the desired raw data and/or computed data for your CAVA analysis."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "85760d97",
+   "metadata": {},
+   "source": [
+    "Below are examples of some of the available selections for the `variable`, `metric_calc`, `ssp_data`, and `export_method` parameters."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b4c35317-e5b9-45bf-9d06-634b0d33725e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Make sure to pass in parameters into the `cava_data` function that are amongst the available options below.\n",
+    "variables = [\"Air Temperature at 2m\", \"Precipitation (total)\", \"NOAA Heat Index\"]\n",
+    "metrics = [\"min\", \"max\", \"mean\", \"median\"]\n",
+    "ssps = [\"SSP2-4.5\", \"SSP3-7.0\", \"SSP5-8.5\"]\n",
+    "export_method = ['off-ramp', 'calculate', 'both']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "234dd10c-b0ee-49a8-929d-a8c692cbfa9a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data = cava_data(\n",
+    "    example_locs.iloc[:2],\n",
+    "    time_start_year=2030,\n",
+    "    time_end_year=2050,\n",
+    "    units=\"degF\",\n",
+    "    downscaling_method=\"Dynamical\",  # default for now ## mandatory\n",
```

Comment:
Statistical doesn’t seem to work yet.

Reply:
Yes, this is intentional for this version. I will be showcasing that this works with Dynamical data and that Statistical data will be included in phase 3 of development.

Reply:
I can add some helper text in the function to make sure the users know this too.